python_modules/dagster/dagster/_core/definitions/repository_definition/repository_data_builder.py

```diff
@@ -219,46 +219,43 @@ def build_caching_repository_data_from_list(
                 )
             # we can only resolve these once we have all assets
             unresolved_jobs[definition.name] = definition
+        elif isinstance(definition, AssetChecksDefinition):
```

Comment:
should this be `has_only_asset_checks`? Or maybe we replace has_only_asset_checks with this instance check?

Reply:
The `isinstance()` here is correct because we are dealing with the definitions the user has passed in directly (we do similar things with `SourceAsset`). `has_only_assets_checks` is for internal use once we are dealing with just `AssetsDefinition`.